frameworks/Crystal/orion/benchmark_config.json

```diff
@@ -0,0 +1,30 @@
+{
+  "framework": "orion",
+  "tests": [
+    {
+      "default": {
+        "json_url": "/json",
+        "plaintext_url": "/plaintext",
+        "db_url": "/db",
+        "query_url": "/queries?queries=",
+        "fortune_url": "/fortunes",
+        "update_url": "/updates?queries=",
+        "port": 8080,
+        "approach": "Realistic",
+        "classification": "Fullstack",
```

Comment:
Classification will be Micro.
A routing lib isn't a fullstack framework.

Reply:
Fixed.
Thanks for letting me know.

Reply:
De nada :+1: